### DISCUSSION TITLE
Built in dreamberd/c function

### DISCUSSION THREAD

**Discussion by GitisAwesome** [TodePond/GulfOfMexico#311] [Ideas]
Why not make a built in function of Dreamberd or C that prints the logo of the language?
Ex:
```  
    _________
   /___   ___\
   | | | |
   | | | |
   | |_| |____
   \_________/
```

**Comment by GitisAwesome** [comment]
Sorry the spaces have been deleted
I added dots to make sure the image kept its shape
Also two slashes wont appear
Thank you.

**Reply by stohrendorf** [reply]
Use inline code using triple backticks "```" in a single line on top and below it, which makes it look like this:
```
this
  conserves
    spaces
```

**Reply by GitisAwesome** [reply]
Thank you I have tried to fix it
